src/Esfa.Vacancy.Register.Infrastructure/Repositories/VacancyRepository.cs

```diff
@@ -56,18 +56,19 @@ public VacancyRepository(IProvideSettings provideSettings)
 ,       V.EmployersWebsite
 ,       V.TrainingTypeId
 ,       RS.LarsCode AS LarsStandardId
-,       CAST(AO.ShortName AS INT) AS SSA1Code
+,       AF.ShortName AS FrameworkCode
 FROM[dbo].[Vacancy]        V
 INNER JOIN [dbo].[VacancyHistory] VH
 	ON V.VacancyId = VH.VacancyId and VH.VacancyHistoryEventSubTypeId = 2
 LEFT JOIN   [Reference].[Standard] RS 
     ON V.StandardId = RS.StandardId
 LEFT JOIN   [Reference].[StandardSector] RSS
     ON RS.StandardSectorId = RSS.StandardSectorId
-LEFT JOIN    [dbo].[ApprenticeshipOccupation] AO
+LEFT JOIN   [dbo].[ApprenticeshipOccupation] AO
```

Comment:
If we are not returning SectorSubjectArea we do not need the joins on Reference.StandardSector or dbo.ApprenticeshipOccupation tables.